### ISSUE TITLE
Changing the legend title

### ISSUE THREAD

**Issue by gioarma** [holoviz/hvplot#430]
#### Problem
When I have a DataFrame `df` with 2 or more columns and plot it with `df.plot()` I get a legend for all my columns and the title of the legend is '**_Variable_**'.

#### Solution I'd like
I would like to have the possibility to set the legend title, or to disable it, because most of the times _'Variable'_ is not the best way to describe the elements in the legend.

I tried to look into the holoviews `opts`, but I didn't find something like `legend_title` or similar, which I think would be the best solution.

Did I miss something? Is there already a way to do this?

**Comment by philippjfr** [comment]
There is indeed setting the `group_label` will change that title.

**Comment by gioarma** [comment]
Yes, thanks! This should definitely be written in the user guide

**Comment by florentbr** [comment]
Is it possible to hide this 'Variable' group_label  ?
Packages like mathplotlib, bokeh and hollowview don't show a group label by defaut.
Is there a reason to display it when it's not even specified in the options ?

**Comment by jbednar** [comment]
I think I agree; the label "Variable" doesn't seem to add anything to what a user can already see is a legend.

**Comment by gioarma** [comment]
I completely agree, I would remove as well the default legend title "Variable"
